### PR TITLE
Configuration for sync process time in base bootstrapper

### PR DESCRIPTION
**Pull request by iulianpascalau**
- configuration value for sync process time used in the base bootstrapper

### CODE REVIEW

File: process/sync/baseSync.go [hunk 1]
```diff
@@ -479,6 +482,9 @@ func checkBootstrapNilParameters(arguments ArgBaseBootstrapper) error {
 	if check.IfNil(arguments.ScheduledTxsExecutionHandler) {
 		return process.ErrNilScheduledTxsExecutionHandler
 	}
+	if arguments.ProcessWaitTime < minimumProcessWaitTime {
+		return fmt.Errorf("%w, minimum id %v, provided is %v", process.ErrInvalidProcessWaitTime, minimumProcessWaitTime, arguments.ProcessWaitTime)
```
Comment (bogdan-rosianu) [inline process/sync/baseSync.go:486]: minimum id? maybe minimum process wait time

Comment (raduchis) [inline process/sync/baseSync.go:486]: minimum **is**

**Review by sasurobert — DISMISSED** [review]

File: process/sync/baseSync.go (hunk 1 above)
Comment (iulianpascalau) [reply process/sync/baseSync.go:486]: :man_facepalming:

Comment (iulianpascalau) [reply process/sync/baseSync.go]: done

Comment (iulianpascalau) [reply process/sync/baseSync.go]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**